src/install/index.js

```diff
@@ -66,7 +70,34 @@ const warmViews = (deployDoc) => {
 
         return updateIndexers(relevantTasks);
       })
-      .then(() => process.stdout.write('.'));
+      .then(() => logIndexersProgress());
+  };
+
+  // logs indexer progress in the console
+  // _design/doc  [||||||||||29%||||||||||_________________________________________________________]
+  const logIndexersProgress = () => {
```

Comment:
maybe `updateIndexers` should return the relevantTasks and then we use that instead of getting them back out of the progress doc? To me seems cleaner, but I could be missing something?